docs/docs/reference/crd-reference/metricsprovider.md

```diff
@@ -34,38 +36,39 @@ spec:
 ```
 
 ## Fields
+<!-- markdownlint-disable MD007 -->
```

Comment:
you always need re-enable the rule  at the end of the section you want to ignore it for